src/Tribe/Ticket_Object.php

```diff
@@ -556,22 +569,37 @@ public function managing_stock( $manages_stock = null ) {
 		 * @return int
 		 */
 		public function inventory() {
+
+			/* @var Tribe__Cache $cache */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 27c74cce08609b48a27c9bd307b193e9</sub>